setup.cfg

```diff
@@ -70,7 +70,7 @@ VCS = git
 style = pep440
 versionfile_source = dandischema/_version.py
 versionfile_build = dandischema/_version.py
-tag_prefix =
+tag_prefix = v
```

Comment:
Try
```suggestion
tag_prefix = [0-9]
```

Reply:
If doesnt work, make it 0 for now

Reply:
That doesn't work, likely due to:

https://github.com/dandi/dandischema/blob/4d9bfbc7943f5607480dee84a6556393cffd20f3/versioneer.py#L630

Reply:
@yarikoptic Setting the prefix to `0` causes the leading `0` to be stripped, resulting in an invalid version string of ".2.0".

Reply:
Oh, no joy... grr...
ok -- let's do a workaround within workflow -- remove all tags which aren't starting with `[0-9]+\.` upon checkout ;-) and I will ask you to fixup versioneer so we could avoid that going forward.

Reply:
I really think it'd be easier to just go with this PR and rename the tags.

Reply:
I dislike all the `v` prefixing in general since taints `git describe` to not be semver right away. Well -- here we would get that even with versioneer fix since we have `schema-` tags, but at least I can `--match` quickly.

But I also dislike  my proposed workaround since it would nohow solve the problem locally.

Awaiting for fix in versioneer is "suboptimal" as well, but here is my thinking "how": https://github.com/python-versioneer/python-versioneer/issues/255 .

Another workaround would be to patch versioneer.py we have to make it work as we like for now at least:  I think the least intrusive would be to keep no prefix, but hardcode `--exclude '[^0-9]*'` or alike to `git-describe` call.  Then whenever versioneer supports it , we would just add `tag_exclude` and happily upgrade our copy of versioneer.  WDYT @jwodder ?

Reply:
Patching our local versioneer to add `--exclude` seems like a workable solution.

Reply:
great, please proceed that way then; here or in #17 so it could turn that CI green ;-)